src/System.Private.CoreLib/shared/System/Text/UTF8Encoding.cs

```diff
@@ -319,20 +466,27 @@ public override unsafe int GetCharCount(byte[] bytes, int index, int count)
         public override unsafe int GetCharCount(byte* bytes, int count)
         {
             // Validate Parameters
+
             if (bytes == null)
```

Comment:
`bytes is null`, while you're here?

Reply:
Same as https://github.com/dotnet/coreclr/pull/21948#discussion_r273688029. Doesn't compile.